README.md

```diff
@@ -26,10 +26,10 @@
 
 | Device | Usage | Price |
 |--------|--------|-------|
-| Activ Buzzer | Buzzer | [1.5 RON](https://www.optimusdigital.ro/ro/audio-buzzere/635-buzzer-activ-de-3-v.html?search_query=buzzer&results=61) |
-| Push Button | Button | [1 RON](https://www.optimusdigital.ro/ro/butoane-i-comutatoare/1119-buton-6x6x6.html?search_query=buton&results=222) |
-| Jumper Wires | Connecting components | [7 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/884-set-fire-tata-tata-40p-10-cm.html?search_query=set+fire&results=110) |
-| Breadboard | Project board | [10 RON](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/8-breadboard-830-points.html?search_query=breadboard&results=145) |
+|Uno R3 ATmega328P| Uno R3 ATmega328P| [45 RON](https://www.emag.ro/placa-dezvoltare-uno-r3-atmega328p-cl201/pd/DPF3WJBBM/?cmpid=87002&utm_source=google&utm_medium=cpc&utm_campaign=(RO:Whoop!)_3P-Y)
+| Activ Buzzer | Buzzer | [2 RON](https://www.emag.ro/buzzer-activ-5v-compatibil-arduino-raspberry-oky0151/pd/D7KJNNMBM/) |
+| Oled| Display| [20 RON](https://www.emag.ro/afisaj-grafic-oled-128-x-64-px-spi-0-96-inch-multicolor-oled-096-spi-white/pd/DPZ798MBM/?cmpid=93116&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_>_Jucarii_hobby&utm_content=111476631565&gad_source=1&gclid=CjwKCAjw3NyxBhBmEiwAyofDYdepSTpZeJRHLUfylAz0D0FFH1Qy-MPPTZx9lRWG9SpZlBKYltj9VxoCex4QAvD_BwE) |
+| Breadboard | Project board | [13 RON](https://www.emag.ro/placa-test-breadboard-830-bb830/pd/D6SCSBMBM/?cmpid=87002&utm_source=google&utm_medium=cpc&utm_campaign=(RO:Whoop!)_3P-Y_>_Jucarii_hobby&utm_content=79559830074&gad_source=1&gclid=CjwKCAjw3NyxBhBmEiwAyofDYTk4lSftqBm5HNeuwBGRkPJC57FY5b366KsxQx3y0N8XTKtlN85YIhoCavAQAvD_BwE) 
 
```

Comment:
Component list is incomplete. You need to add all components that are shown in the schematic. (add joysticks)